### PR TITLE
Grab local image from Env if available

### PR DESCRIPTION
**Pull request by chainlink**
Enables tests to run based on a passed in SHA

### CODE REVIEW

File: src/internal/minikubetestenv/deploy.go [hunk 1]
```diff
@@ -69,6 +69,15 @@ type DeployOpts struct {
 
 type helmPutE func(t terraTest.TestingT, options *helm.Options, chart string, releaseName string) error
 
+func getLocalImage() string {
+	sha := os.Getenv("TESTSHA")
```
Comment (acohen4) [inline src/internal/minikubetestenv/deploy.go:73]: super nit: WDYT about changing env var to `TEST_IMAGE_SHA`?

**Review by acohen4 — APPROVED** [review]
LGTM

File: src/internal/minikubetestenv/deploy.go [hunk 2]
```diff
@@ -69,6 +69,15 @@ type DeployOpts struct {
 
 type helmPutE func(t terraTest.TestingT, options *helm.Options, chart string, releaseName string) error
 
+func getLocalImage() string {
+	sha := os.Getenv("TEST_IMAGE_SHA")
```
Comment (jrockway) [inline src/internal/minikubetestenv/deploy.go:73]: Best style here is to keep the variable binding in the if statement, and to not indent the "default":

```go
if sha := os.Getenv("TEST_IMAGE_SHA"); sha != "" {
    return sha
}
return localImage
```

**Review by jrockway — APPROVED** [review]
Good idea!

**Pull request merged**